docs/UserGuide.md

```diff
@@ -49,18 +49,23 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
 
 ### Installation
 
-1. Ensure that you have [Java 11 or above](https://www.java.com/en/download/) installed on your computer.
-    - If you are a MacOS, follow the instructions [here](https://nus-cs2103-ay2324s2.github.io/website/admin/programmingLanguages.html) instead.
+1. Ensure that you have **Java 11 or above** installed on your computer.
+    - For MacOS, follow the instructions [here](https://nus-cs2103-ay2324s2.github.io/website/admin/programmingLanguages.html).
+    - For Windows or Linux, follow the instructions [here](https://www.java.com/en/download/). 
 2. Download the latest `CareerSync.jar` file [here](https://github.com/AY2324S2-CS2103T-W11-1/tp/releases/tag/v1.2).
```

Comment:
Actually navigating to our release page, our jar file is in lowercase. Should this be lowercase too, or the release name changed to CamelCase for consistency

Reply:
CareerSync should be kept it in camel case, the jar file was auto-generated by gradle